scout/server/blueprints/variants/controllers.py

```diff
@@ -40,6 +40,7 @@
     update_variant_case_panels,
 )
 from scout.server.blueprints.variants.forms import BetterDecimalField
+from scout.server.extensions import store
```

Comment:
Just note that you already import it in the view. I don't think it matters here though, at the very least not now.

Reply:
I know I know. The thing is why do we pass around all these adapters from views when they are also directly accessible on controllers? I'd move towards refactoring in here, so we don't have to pass so many params around